File: res/OtherSource.py
class OtherSource:

  def __init__(self, ):
    pass

  def main(args):
    os = OtherSource()
    demo = '{}{}{}{}'.format("uno: ", 1, "; dos: ", 2)
    print(demo)

import sys
if __name__ == '__main__':
  OtherSource.main(sys.argv)

File: in/OtherSource.java
public class OtherSource {

    public static void main(String[] args) {
        OtherSource os = new OtherSource();
        for (int i = 2; i < 500 && i%1==0; i = i*i) {
            System.out.println(i/2);
        }
    }
}